modules/bootconf.py

```diff
@@ -14,11 +14,16 @@
 
 def configureBootloader(old, new, conffile):
     ''' Configure bootloader to use the updated rootfs '''
-    if runningDevice(conffile) == 'raspberry-pi' or runningDevice(conffile) == 'raspberry-pi2':
+    if runningDevice(conffile) in ['raspberry-pi', 'raspberry-pi2', 'raspberrypi3']:
         b = BCMRasberryPiBootloader(conffile)
         if not b.configure(old, new):
             log.error("Could not configure bootloader.")
             return False
+    elif runningDevice(conffile) == 'intel-nuc':
```

Comment:
Can you store `runningDevice(conffile)` in a var, rather than calling it multiple times